src/Controller/AdminController.php

```diff
@@ -61,7 +61,7 @@ protected function createSearchQueryBuilder(
         // Use only list fields in search query.
         $this->entity['search']['fields'] = array_filter($this->entity['search']['fields'], function ($key) {
             return isset($this->entity['list']['fields'][$key]);
-        }, ARRAY_FILTER_USE_KEY);
+        }, \ARRAY_FILTER_USE_KEY);
```

Comment:
Is this `\` needed?

Reply:
Not really, but the coding standards tells me to add it. And I always do what I'm told!